src/pages/sa-trx/[functionId]/forms/SaleTransactionForm.js

```diff
@@ -384,6 +400,13 @@ export default function SaleTransactionForm({ labels, access, recordId, function
           const basePriceValue = postMetalToFinancials === false ? basePrice : 0
           const TotPricePerG = basePriceValue
 
+          const unitPrice =
```

Comment:
below in update it's enough to pass unitPrice, don't repeat the whole condition again